### PR TITLE
Fix nominate and owner scripts

### PR DESCRIPTION
**Pull request by leomassazza**
Nominate and Owner scripts fails if any contract in config.json is not deployed (or present in deployment.json).
This PR adds a check on that condition and show a warning while trying to Nominate or Accept ownership.

### CODE REVIEW

File: publish/src/commands/nominate.js [hunk 1]
```diff
@@ -110,7 +110,14 @@ const nominate = async ({
 		}
 	}
 
+	const warnings = [];
```
Comment (leomassazza) [inline publish/src/commands/nominate.js:113]: Adding this to be able to show a summary at the end of the process with all missing contracts found.

File: publish/src/commands/owner.js [hunk 2]
```diff
@@ -242,7 +242,14 @@ const owner = async ({
 	}
 
 	console.log(gray('Looking for contracts whose ownership we should accept'));
+	const warnings = [];
```
Comment (leomassazza) [inline publish/src/commands/owner.js:245]: dito

File: publish/src/commands/connect-bridge.js [hunk 3]
```diff
@@ -238,7 +238,7 @@ const setupInstance = async ({
 		useFork,
 		useOvm,
 	});
-	console.log(gray('  > provider:', provider.url));
+	console.log(gray('  > provider:', provider.connection.url));
```
Comment (leomassazza) [inline publish/src/commands/connect-bridge.js:241]: small fix. Provider was shown as undefined because of that missing intermediate object

Comment (ajsantander) [reply publish/src/commands/connect-bridge.js:241]: This must be a change in the ethers interface

**Review by ajsantander — APPROVED** [review]

**Pull request merged**